### PR TITLE
Helper script to update and templatize an appliance.

### PR DESCRIPTION
**Pull request by mfalesni**
Use it like this:
`scripts/update_template.sh providerkey appliancename -u onerepository -u secondrepository`

When this gets merged, I will probably point sprout to the templatize method since the code there is basically copied from sprout.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 56c02b0ceae74505786d8c858a464f0696b14361:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: utils/appliance.py [hunk 1]
```diff
@@ -275,6 +275,43 @@ def destroy(self):
...
+        self.provider.stop_vm(self.vm_name)
+        self.provider.wait_vm_stopped(self.vm_name)
+
+    def start(self):
+        """Starts the VM this appliance is running as
+        """
+        self.provider.start_vm(self.vm_name)
+        self.provider.wait_vm_running(self.vm_name)
+
+    def templatize(self):
```
Comment (psav) [inline utils/appliance.py]: I would like to see this split up across the two classes, so the "cleaning" is done in IPAppliance, but the templatize is done in Appliance.

**Comment by dajoRH** [comment]
Lint report for commit eec8ad748f990eb21f15937558fc3597055bb851:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: utils/appliance.py (hunk 1 above)
Comment (mfalesni) [inline utils/appliance.py]: Moved

**Comment by dajoRH** [comment]
Lint report for commit 870a94f1b1a4658e5c5ec605b989a04c2080f96e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**